commit 902d68a65813ecdd092d553775a40eac243197a4
Author: Trent Mick <trentm@gmail.com>
Date:   2019-03-05T16:17:11-08:00 (7 months ago)
    
    TRITON-1286 test-amon failure in nightly-2: maintenances.test.js: timeout in "maint 2: stop amontestzone"
    Reviewed by: Chris Burroughs <chris.burroughs@joyent.com>
    Approved by: Chris Burroughs <chris.burroughs@joyent.com>

diff --git a/test/maintenances.test.js b/test/maintenances.test.js
index acc9f45..2403881 100644
--- a/test/maintenances.test.js
+++ b/test/maintenances.test.js
@@ -276,7 +276,7 @@ var maint1AlarmId;
 
 test('maint 1: stop amontestzone', {timeout: TIMEOUT_MAX}, function (t) {
     notifications = []; // reset
-    common.vmStop({uuid: prep.amontestzone.uuid, timeout: 3 * 60 * 1000},
+    common.vmStop({uuid: prep.amontestzone.uuid, timeout: 10 * 60 * 1000},
         function (err) {
             t.ifError(err, 'stopped amontestzone');
             t.end();
@@ -451,7 +451,7 @@ test('maint 2: create maint window', function (t) {
 test('maint 2: stop amontestzone', {timeout: TIMEOUT_MAX}, function (t) {
     notifications = []; // reset
     // I've seen this VM stop on nightly-2 take >1 min.
-    common.vmStop({uuid: prep.amontestzone.uuid, timeout: 3 * 60 * 1000},
+    common.vmStop({uuid: prep.amontestzone.uuid, timeout: 10 * 60 * 1000},
         function (err) {
             t.ifError(err, 'stopped amontestzone');
             t.end();
